gems/activesupport/6.0/activesupport-generated.rbs

```diff
@@ -5609,9 +5609,9 @@ class ERB
     #
     #   puts html_escape('is a > 0 & a < 10?')
     #   # => is a &gt; 0 &amp; a &lt; 10?
-    def self?.html_escape: (untyped s) -> untyped
+    def self?.html_escape: ...
```

Comment:
Technically, we don't need the `def` here. Adding the `def` syntax to tell an implementation is added by activesupport.